src/components/player/download-control.tsx

```diff
@@ -84,24 +101,11 @@ const DownloadControl: FunctionComponent<
 > = ({slug, download_url, state}) => {
   return (
     <DownloadControlErrorBoundary>
-      <Tippy
-        offset={[0, -2]}
-        content={
-          <div className="px-2 py-1 text-sm bg-gray-900 rounded-sm">
-            {download_url
-              ? 'Download video'
-              : 'Become a member to download this lesson'}
-          </div>
-        }
-      >
-        <div>
-          <DownloadButton
-            slug={slug}
-            download_url={download_url}
-            state={state}
-          />
-        </div>
-      </Tippy>
+      <DownloadButtonWithTooltip
+        slug={slug}
+        download_url={download_url}
+        state={state}
+      />
```

Comment:
put tooltip logic in the button component. Wrapping the functional component in `<Tooltip.Trigger>` had a refs issue that I wasn't quite sure how to fix.